docs/modules/reference/pages/glossary.adoc

```diff
@@ -73,16 +98,25 @@ Use to generate requisition XML files for creating/updating/removing nodes for m
 Provisioning:: The process of getting your devices, applications, and services into monitoring.
 (See xref:operation:provisioning/introduction.adoc#discovery-auto[Provisioning].)
 
-Remote Method Invocation (RMI):: Java API that lets one Java Virtual Machine (JVM) running object to invoke methods on an object running in another JVM. RMI integration lets you access Horizon/Meridian through jconsole or remote manage them.
+Remote Method Invocation (RMI):: Java API that lets one Java Virtual Machine (JVM) running object to invoke methods on an object running in another JVM. 
+RMI integration lets you access Horizon/Meridian through jconsole or remote manage them.
 (See xref:operation:admin/rmi.adoc#rmi[Enabling RMI].)
 
+Requisitions:: Sets of nodes to import into {page-component-title} for monitoring and management.
+Requisitions can be iteratively built and imported at a later date.
+(See xref:operation:provisioning/getting-started.adoc#requisition-create[Create a Requisition].)
+
 Round Robin Database (RRD):: Special type of database designed for collection, visualization, and analysis of time-series data.
 Data is maintained as a fixed-size circular-buffer that overwrites the oldest data with new data.
 
 Sentinel:: Karaf container that provides scalability for data processing, including flows, SNMP traps, syslog messages, and streaming telemetry in OpenNMS.
 It also supports thresholding for streaming telemetry if you are using OpenNMS Newts.
 (See xref:deployment:sentinel/introduction.adoc#sentinel[Sentinel].)
 
+Snaps:: Self-contained software packages that run in a sandbox and have mediated access to host systems.
+OpenNMS uses snaps to distribute platform binaries, upgrades, and initiate rollbacks.
+(See https://dev.cloud.opennms.com/docs/opennms-cloud/welcome/introduction.html?q=snaps#minion-architecture[OpenNMS Appliance Service - Architecture].)
```

Comment:
Is the `dev.cloud.opennms.com` URL meant to be linked for public use at this time?
```suggestion
Our Appliance Service uses snaps to distribute operating system packages and Minion service updates.
(See https://dev.cloud.opennms.com/docs/opennms-cloud/welcome/introduction.html#_software_management[Appliance Software Management].)
```

Reply:
@Bonrob2, could you confirm this? I did find the info on the dev.cloud.opennms.com site, but can remove the reference for now if necessary.

Reply:
Plus, this is only relevant in the context of the Appliance Service since snaps aren't part of Horizon/Meridian at this time.